socorro/collector/wsgi_breakpad_collector.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, config):
         self.accept_submitted_legacy_processing = \
             self._get_accept_submitted_legacy_processing()
         self.throttler = self._get_throttler()
+        self.metrics = self._get_metrics()
```

Comment:
Why not just `self.metrics = self.config.metrics`?
What's the purpose of the `_get_metrics` method?